### DISCUSSION TITLE
Prisma 2 deployment not working for Heroku.

### DISCUSSION THREAD

**Discussion by utkarshchowdhary** [prisma/prisma#4109] [Q&A] [answered]
I had create a graphql api using prisma orm a month back tested and deployed on heroku was working fine then. But recently when i was reviewing the live hosted url of heroku it wasn't working anymore (show server error when making any request related to accessing data throught prisma from database) but it's woking on my localhost.

Here is the link for my repo **https://github.com/utkarshkororo/prisma-blouge**.
Any suggestion would be helpful.

**Comment by ryands17** [comment]
Hey @utkarshkororo 👋 
Could you check [this](https://www.prisma.io/docs/guides/deployment/deploying-to-heroku#overview) tutorial and see if all the steps that you have followed are correct?

**Reply by utkarshchowdhary** [reply, marked as answer]
Hi @ryands17 thx for responding turns out there was some issue with the node version 12.19, I was using 12.16 on my local so when I switched it to 12.16 on heroku as well it was working again,  I was using 12.19 before.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
